localStorage_value_checker_sample_app/app.js

```diff
@@ -0,0 +1,43 @@
+(function() {
+  return {
+    events: {
+      // Framework Events
+      'app.created':'checkLocalStorage', // Call 'checkLocalStorage' function below
+      // Request Events (None)
+      // DOM events
+      'keyup #inputValueId': function(event){ // User clicked the 'enter'/'return' button so store value in localStorage
+        if(event.keyCode === 13)
+          return this.processInputValue();
+      },
+      'click .reset': 'resetValue' // Call 'resetValue' function below
+    },
+    checkLocalStorage: function () { // Look in localStorage for if value is present for 'exampleString'
+        if (this.store('exampleString') === null) { // If no value
+          services.notify('<span style="font-size: 14px;">localStorage value not set</span>', 'alert', 8000); // Services notification
+          this.switchTo('inputValue'); // Display input field for user to set value of localStorage
+        } else { // There is a value already set for 'exampleString' in localStorage
+          services.notify('<span style="font-size: 14px;">localStorage value found</span>', 'notice'); // Services notification
```

Comment:
you could use the same template as the one on line 16, and just have it pass in a message. also better to use a class on the span and put the styles in the CSS file than to inline the styles.